druid/src/widget/split.rs

```diff
@@ -27,8 +27,10 @@ pub struct Split<T> {
     solid: bool,
     draggable: bool,
     min_size: f64,
-    split_point: f64,
+    split_point_chosen: f64,
+    split_point_effective: f64,
     splitter_size: f64,
+    min_splitter_area: f64,
```

Comment:
Maybe this should be an `Option<f64>`, so that we can default to using the splitter size when this isn't passed explicitly? Maybe?

I'm not sure. I'm just thinking of in the example where you set the splitter size to 3, should we keep the same default min_splitter_area? Maybe...

Reply:
I think having it set by default with some reasonable value is best. The main reason this exists is to allow extremely narrow splitter bars without sacrificing the user experience by requiring them to pixel hunt. So if someone wants a 3px bar, or even a 1px bar, they can easily do so but druid will default to a 6px bar area. If they want the area to also be 1px then they can set the minimum area to e.g. 0px.